src/components/student-dashboard/progress-guide.cjsx

```diff
@@ -94,9 +94,6 @@ ProgressGuidePanels = React.createClass
 
         <ProgressGuide sections={recent} courseId={@props.courseId} />
 
-        <PracticeButton ref='practiceBtn' title='Practice my weakest topics'
-            courseId={@props.courseId} sections={practiceSections} />
-
         <BS.Button
```

Comment:
Is the `practiceSections` variable used now that the button is no more?  I suspect that we might be able to remove it from line 88-90 above.  Can we remove the `require` for the `PracticeButton` component as well?